### ISSUE TITLE
[Meds / Accessibility] Fix headings in accordion with incorrect heading structure

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#125254]
### 📝 User story

As a screen reader user, I want correctly identified heading elements, so that I can use heading navigation and the heading list feature to understand and navigate information structures.

### 📝 Problem statement

Headings in the accordion component are all provided has level 4 (`<h4>`) headings.

### 🌌 Supporting screenshots or videos

<img width="547" height="618" alt="Screenshot of accordion component with an overlay showing heading levels" src="https://github.com/user-attachments/assets/fcffc067-1c61-4964-85ac-9d90f60fa50e" />

Headings in accordion controls are `<h4>`s and subheadings within accordion panels are also `<h4>`s.

### 👣 Reproduction steps

1. Navigate to the [Medications detail page](https://staging.va.gov/my-health/medications/prescription/24654491) (username: (mhvmark.test@id.me).
2. Use the ARC Toolkit browser extension to show headings.
3. Notice that the Accordion headings are all `<h4>`s.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure.](https://www.w3.org/WAI/WCAG22/Understanding/adaptable)
  - [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks.](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps

Change the subheadings in the accordion component to `<h5>`s.

### ✅ Acceptance criteria

- [ ] Change is reflected in Figma
- [ ]  The system displays the accordion control headings as level 4 headings and headings in the content of accordion panels as level 5 headings.
- [ ] The screen reader user can navigate by heading level and can understand the information structure of the accordion controls and panels.

### ✅ Possible tasks

- [x] Review, refine, and acknowledge feedback.
- [x] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by michaelbrodsky-a6** [comment]
Thanks so much @sarahhortonadhoc – really appreciate your comment. All makes sense! We've officially picked up the ticket and have begun moving forward with next steps.
